python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -1201,6 +1201,12 @@ interface {{ ethernet_interface.name }}
    {{ auth_failure_fallback_mba }}
 {%         endif %}
 {%     endif %}
+{%     if ethernet_interface.traffic_engineering.enabled is arista.avd.defined(true) %}
```

Comment:
The order is not correct as per EOS. It should come between sync-e and `link tracking group`

Reply:
You are right, corrected this for ethernets and port-channels.